packages/cluster-ui/src/statementsPage/statementsPage.tsx

```diff
@@ -306,6 +320,25 @@ export class StatementsPage extends React.Component<
       .filter(
         statement =>
           sqlTypes.length == 0 || sqlTypes.includes(statement.stats.sql_type),
+      )
+      .filter(
+        statement =>
+          regions.length == 0 ||
+          containAny(
+            statement.stats.nodes.map(
+              node => nodeRegions[node.toString()],
+              regions,
+            ),
+            regions,
+          ),
+      )
+      .filter(
+        statement =>
+          nodes.length == 0 ||
+          containAny(
+            statement.stats.nodes.map(node => "n" + node),
+            nodes,
+          ),
```

Comment:
I like these pithy functional statements quite a lot but it would also be nice to perhaps have an explanatory comment somewhere summarizing what filtering is taking place and for what purpose.  This will help make it easier to extend or modify in the future.

Reply:
done